speech/grpc/transcribe_streaming.py

```diff
@@ -120,14 +116,13 @@ def record_audio(rate, chunk):
     # This is necessary so that the input device's buffer doesn't overflow
     # while the calling thread makes network requests, etc.
     fill_buffer_thread = threading.Thread(
-        target=_fill_buffer, args=(audio_stream, buff, chunk))
+        target=_fill_buffer, args=(audio_stream, buff, chunk, stoprequest))
     fill_buffer_thread.start()
 
     yield _audio_data_generator(buff)
 
-    audio_stream.stop_stream()
```

Comment:
Any particular reason not to stop the stream before closing? While I'm not certain it's necessary, [all the examples](https://people.csail.mit.edu/hubert/pyaudio/docs/) stop the stream before closing it.